Cargo.toml

```diff
@@ -15,7 +15,12 @@ keywords = ["OpenEXR"]
 [dependencies]
 libc = "0.2"
 half = "1"
+clippy = { version = "*", optional = true }
 
 [dependencies.openexr-sys]
 path = "openexr-sys"
 version = "0.5.0"
+
+[features]
+unstable = ["clippy"]
```

Comment:
When/how is the `unstable` feature enabled?

Reply:
Ah sorry, I forgot I didn't reply to this. It's enabled automatically by travis-cargo on nightly builds